### PR TITLE
Deemphasizes workflow in prominence in the docs

### PR DESCRIPTION
**Pull request by snide**
This is a draft PR for discussion with @richardgill / @kostasb / @alexfrancoeur 

Our workflow has limitations that make difficult to lead with as a feature offering. Within the docs, this PR:

- Removes workflow's prominence in our docs, and adds an alert to the top of the workflow documentation
- Moves the available regions page outside of getting started, and moves it to resources (for now).

No slugs are changed. URLs will continue to work.

Is paired with https://github.com/xataio/frontend-next/pull/2921 on the website

### CODE REVIEW

File: 080-Recipes/040-workflow.mdx [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
 slug: getting-started/workflow
 published: true
 ---
 
+<Alert status="warning">
+  **This feature is in a beta state**. We are currently rebuilding our migration strategy to incorporate
+  [pgroll](https://github.com/xataio/pgroll) alongside native Postgres support. Until then, migrations and branching as
+  documented below are limited in scope. Specifically, the GitHub integration does not work within a monorepo with
+  multiple Xata databases, and File attachments within records will not get copied to generated preview branches.
+</Alert>
```
Comment (kostasb) [inline 080-Recipes/040-workflow.mdx:15]:
```suggestion
<Alert status="warning">
  **This feature is in a beta state**. We are currently rebuilding our migration strategy to incorporate
  [pgroll](https://github.com/xataio/pgroll) alongside native Postgres support. Until then, migrations and branching as
  documented below are limited in scope. Specifically, the GitHub integration does not work within a monorepo with
  multiple Xata databases, File attachments within records will not get copied to generated preview branches, there is no rebase capability so the migration changes must be linear and Pull Requests should be issued to the main branch.
</Alert>
```

**Comment by richardgill** [comment]
Screenshot 2024-02-07 at 09 07 50
@snide should we also remove it from here?

File: 080-Recipes/040-workflow.mdx [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
-navTitle: Branching workflow
+navTitle: GitHub branching workflow
 keywords: ['Vercel', 'GitHub', 'branch', 'Netlify']
 description: Learn about core the Xata workflow, branching, and migrations using GitHub with Netlify or Vercel
 slug: getting-started/workflow
 published: true
 ---
 
+<Alert status="warning">
+  **This feature is in a beta state**. We are currently rebuilding our migration strategy to incorporate
```
Comment (richardgill) [inline 080-Recipes/040-workflow.mdx:11]:
```suggestion
  **This feature is in beta**. We are currently rebuilding our migration strategy to incorporate
```

Nit suggestion

File: 080-Recipes/040-workflow.mdx (hunk 1 above)
Comment (snide) [reply 080-Recipes/040-workflow.mdx]: Reworded a bit, but included your additions.

File: 080-Recipes/040-workflow.mdx (hunk 2 above)
Comment (snide) [reply 080-Recipes/040-workflow.mdx]: Fixed in separate commit.

**Review by richardgill — APPROVED** [review]
LGTM

**Pull request merged**